docs/egress.md

```diff
@@ -0,0 +1,297 @@
+# Egress
+
+## Table of Contents
+
+<!-- toc -->
+- [What is Egress?](#what-is-egress)
+- [Prerequisites](#prerequisites)
+- [The Egress resource](#the-egress-resource)
+  - [AppliedTo](#appliedto)
+  - [EgressIP](#egressip)
+  - [ExternalIPPool](#externalippool)
+- [The ExternalIPPool resource](#the-externalippool-resource)
+  - [IPRanges](#ipranges)
+  - [nodeSelector](#nodeselector)
+- [Usage examples](#usage-examples)
+  - [Configuring high available Egress](#configuring-high-available-egress)
+  - [Configuring static Egress](#configuring-static-egress)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## What is Egress?
+
+`Egress` is a CRD API that manages external access from the Pods in a cluster.
+It supports specifying which egress (SNAT) IP the traffic from the selected Pods
+to the external network should use. When a selected Pod accesses the external
+network, the egress traffic will be tunneled to the Node that hosts the egress
+IP if it's different from the Node that the Pod runs on and will be SNATed to
+the egress IP when leaving that Node.
+
+You may be interested in using this capability if any of the below are true:
+
+- A consistent IP address is desired when specific Pods connect to services
+  outside of the cluster, for source tracing in audit logs, or whitelisting
+  by source IP in external firewall, etc.
+
+- You want to limit outgoing external connections leave the cluster via certain
```

Comment:
```suggestion
- You want to limit outgoing cluster-external connections via certain
```

Reply:
Does "limit outgoing cluster-external connections via certain Nodes" have the same meaning? 
What do you think about the latest one: "force outgoing external connections to leave the cluster via certain Nodes"?